.golangci.yaml

```diff
@@ -12,7 +12,7 @@ linters-settings:
       - 'github.com/jackc/pgx'
   gosec:
     excludes:
-      - 'G404' # Allow the usage of math/rand
+      - 'G404'  # Allow the usage of math/rand
```

Comment:
is there a way we can turn this lint rule off? VSCode auto-formats it away and its a constant pain

Reply:
I can look

Reply:
I don't see one, unless I'm mising it? https://yamllint.readthedocs.io/en/stable/rules.html

Reply:
D'oh